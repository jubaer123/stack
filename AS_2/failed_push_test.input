u 6
u 98
u 65
u 284
u 1
u 0
x