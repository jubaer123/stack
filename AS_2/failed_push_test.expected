6
98
65
284
1
failed push
Successfully executed 5 instructions
|_ 6 _|
|_ 98 _|
|_ 65 _|
|_ 284 _|
|_ 1 _|
5 elements
